### PR TITLE
Add download capability for workflow run attempt logs

### PR DESCRIPTION
**Pull request by btacyildiz**
Fixes: #2715

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you, @btacyildiz !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by valbeat — APPROVED** [review]
LGTM!

**Comment by gmlewis** [comment]
Thank you, @valbeat !
Merging.

**Pull request merged**
